### PR TITLE
chore: google analytics tag rationalisation - one each for prod and dev

### PR DESCRIPTION
**Pull request by algreasley**
... and a good shake-down of the server dependencies!

### CODE REVIEW

File: client/src/index.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ async function init() {
   serviceWorker.register()
 
   window.gtag("js", new Date())
-  window.gtag("config", env === "demo" ? "G-HHXS2LJP6P" : "G-Y28QSEPEC8")
+  window.gtag("config", env === "demo" ? "G-Z3PC9MRCH9" : "G-Y28QSEPEC8")
```
Comment (SHolleworth) [inline client/src/index.tsx:30]: should they both be `demo` or both be `prod`?

Comment (algreasley) [reply client/src/index.tsx:30]: fair point .. I was fitting in with previous designation (from getEnvironment)

File: client/vite.config.ts [hunk 2]
```diff
@@ -48,7 +48,7 @@ const injectScriptIntoHtml = (env: string) =>
         injectScript: `
           <link rel="manifest" href="/manifest.json" />
           <script async src="https://www.googletagmanager.com/gtag/js?id=${
-            env === "demo" ? "G-HHXS2LJP6P" : "G-Y28QSEPEC8"
+            env === "prod" ? "G-Z3PC9MRCH9" : "G-Y28QSEPEC8"
```
Comment (algreasley) [inline client/vite.config.ts:51]: blast .. this should not have changed .. copied over from RT (damn this prod / demo dichotomy)

File: client/src/index.tsx (hunk 1 above)
Comment (algreasley) [reply client/src/index.tsx:30]: i.e. demo is "correct" - "prod" is desired, per RT but we are not there yet.

**Review by SHolleworth — APPROVED** [review]

**Pull request merged**